### ISSUE TITLE
[WIP] Chapter: IDEs, editors and notebooks

### ISSUE THREAD

**Issue by NatalieThurlby** [alan-turing-institute/the-turing-way#617]
## Detailed description
 * This issue is a place to discuss matters relating to the writing of the chapter on _IDEs, editors and notebooks_.
 * This chapter is being written in #(will edit soon)
 * This chapter is being developed on the master branch.

### Table of contents
To-do.

## Current status
I began writing a chapter outline. If anyone would like to contribute, please do!

 - [ ] Write chapter outline
 - [ ] Add material to the chapter
 - [ ] Combine materials into a readable chapter
 - [ ] Proofread
 - [ ] Request reviews
 - [ ] Address reviews
 - [ ] Merge to master branch.

## Updates
None yet.

**Comment by mjg211** [comment]
Very keen to help with this one 😃

**Comment by alexmorley** [comment]
Yeah I'm gonna tag myself in this thread too. Happy to chat about notebooks especially and have done a bit of research on how people use notebooks as IDEs (and what practices make that easier :) )

**Comment by NatalieThurlby** [comment]
Yay - glad people are interested 😃. I really only got the last 45 minutes of the book dash to think about this, so the outline is very basic, but I'll make my PR #619 Ready to Review so you guys can make a start.(handing in my thesis in just over a week, so will be of zero use until after that).

**Comment by harisood** [comment]
Hello from autumn 2022 @NatalieZelenka! Hope all is well with you, many months later. The TTW Core team is following up on issues in our backlog and wanted to check if we now should close this issue? If so, feel free to comment or 👍  this comment. If not, please leave a comment on how we could revive this thread and how we can support this work.

We'll leave the issue open for now, and if we don't hear from you within a few weeks we'll mark it as closed - but we are always happy to reopen it! Thank you!

**Comment by NatalieZelenka** [comment]
I think there was some worry that including this kind of content might be too opinionated about tools or imply that you can't work reproducibly in some tools, which is why I didn't continue working on this. Sorry that I didn't document that at the time. Please do close it if that's still the current thought process. 

Or if it is a topic that would be appropriate, then Ill see if I can set aside some time to tidy up what I started on and submit an updated PR.

 Let me know!

**Comment by dingaaling** [comment]
Thanks for the follow up @NatalieZelenka - if you have the time to submit a PR on the work you started here, we would be interested in seeing and reviewing it! For now I'll leave this issue open and please let us know how we can support you with this!
